js/main.js

```diff
@@ -1,50 +1,5 @@
-const COMMENTS_NAMES = ['James', 'Julia', 'Alex', 'Michel', 'Grag', 'John'];
-const MESSAGES = ['Всё отлично!', 'В целом всё неплохо. Но не всё.', 'Когда вы делаете фотографию, хорошо бы убирать палец из кадра. В конце концов это просто непрофессионально.',
-  'Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.', 'Я поскользнулся на банановой кожуре и уронил фотоаппарат на кота и у меня получилась фотография лучше.',
-  'Лица у людей на фотке перекошены, как будто их избивают.', 'Как можно было поймать такой неудачный момент?!'];
-const DESCRIPTIONS = ['Хороший ракурс', 'Отметили так отметили', 'No comments', 'Закат на Балли', 'Утро в Сочи', 'Милая фотка', 'Огонь'];
-const MAX_POST = 25;
-const AVATAR_SRC = 'img/avatar-';
-const AVATAR_FORMAT = '.svg';
-const PHOTO_SRC = 'photos/';
-const PHOTO_FORMAT = '.jpg';
-const MIN_LIKE = 15;
-const MAX_LIKE = 200;
+import {MAX_POST} from './render-moks-data';
+import {getArrayMocks} from './render-moks-data';
```

Comment:
Всё что мы импортируем из одного модуля нужно записывать внутри одних фигурных скобочек и через запятую, чтобы данные строкой ниже не перезатирали то что выше:
```
import { MAX_POST, getArrayMocks } from './render-moks-data';
```

А еще забыл указать расширение файла, так что наверное на это браузер и ругался, т.к. думал что это ты у него такое запрашиваешь 😉 
```suggestion
import {MAX_POST, getArrayMocks} from './render-moks-data.js';
```